cross/libusb/Makefile

```diff
@@ -1,16 +1,27 @@
 PKG_NAME = libusb
 PKG_VERS = 1.0.9
-PKG_EXT = tar.bz2
-PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://sourceforge.net/projects/libusb/files/libusb-1.0/libusb-1.0.9
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/libusb/libusb/archive
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
 DEPENDS =
 
-HOMEPAGE = http://sourceforge.net/projects/libusb/
-COMMENT  = Library to enable user space application programs to communicate with USB devices
-LICENSE  =
+HOMEPAGE = https://libusb.info/
+COMMENT  = C library that provides generic access to USB devices. It is intended to be used by developers to facilitate the production of applications that communicate with USB hardware.
+LICENSE  = LGPL-2.1
 
 GNU_CONFIGURE = 1
+PRE_CONFIGURE_TARGET = libusb_pre_configure
 
 include ../../mk/spksrc.cross-cc.mk
+
+
+ifeq ($(findstring $(ARCH), $(ARM_ARCHES)),$(ARCH))
+libusb_pre_configure:
+	$(RUN) ; ./autogen.sh --host=arm-linux
```

Comment:
Please set a variable - for instance LIBUSB_AUTOGEN_PARAMS - to avoid "libusb_pre_configure" target duplicates.